src/request/export/ExportWords.js

```diff
@@ -19,7 +20,7 @@ class ExportWords extends Nimiq.Observable {
      * if a complete page is missing it will be created.
      * However these pages wil be the default pages which usually don't match the applications requirements.
      * Refer to the corresponsing _build(Privcy | RecoveryWords | ValidateWords) to see the general Structure.
-     * @param {Parsed<KeyguardRequest.SimpleRequest>} request
+     * @param {Parsed<KeyguardRequest.ExportRequest>} request
```

Comment:
Is an extra type `ExportRequest` now still required here or is that just a leftover from when the `useLegacyStore` flag had to be provided by the caller?

Reply:
It would be good to have an extra type, yes, for the reasons in the comment below.

Reply:
The ExportRequest also has the `fileOnly` and `wordsOnly` flags, so it was actually a mistake that it was a SimpleRequest here before.